### PR TITLE
chore: add volunteer chozeur

### PR DESCRIPTION
**Pull request by chozeur**
Hi all, glad to take part in this project !

### CODE REVIEW

File: data/users/chozeur.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"name": "ChoZeur",
+	"bio": "Willing dev, student at 42 Paris",
```
Comment (codewithvoid) [inline data/users/chozeur.json:3]: Use title here. Like Sofware Developer / Software Engineer.

That will be useful for developers seeking help.

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @chozeur. Added a somment. PTAL.

File: data/users/chozeur.json (hunk 1 above)
Comment (chozeur) [reply data/users/chozeur.json]: My bad, I've made the changes. Thanks.

File: data/users/chozeur.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+	"name": "ChoZeur",
+	"bio": "Sofware Developer",
```
Comment (codewithvoid) [inline data/users/chozeur.json:3]: fix typo :)

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: data/users/chozeur.json (hunk 2 above)
Comment (chozeur) [reply data/users/chozeur.json]: my bad

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome!

You can join the community [here](https://discord.gg/Tn9E5pn5)

**Pull request merged**